.github/workflows/release_gem_package.yml

```diff
@@ -0,0 +1,38 @@
+# Reference Workflow: https://github.com/actions/starter-workflows/blob/main/ci/gem-push.yml
+name: Release Gem Package
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'lib/datum/version.rb'
+
+jobs:
+  build:
+
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create a new tag based on the latest version
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
+          sh ./release_package.sh
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.5.0
```

Comment:
Where does this version come from?

Reply:
I couldn't find it specified on the repo so ended up calling `ruby -v`
```
Kathys-MacBook-Pro:datum kc$ ruby -v
ruby 2.5.0p0 (2017-12-25 revision 61468) [x86_64-darwin18]
```

Reply:
Won't that give you whatever you have set for rbenv global if nothing is set for local? Should I do the same if a version is not set? Our latest version is ruby 3

Reply:
oh yeah I forgot about rbenv global
@jessie19 Do you have any input on this? Since I know datum hasn't been updated in a long time

Reply:
Let's stick with ruby 2.5 for now.

If we like to upgrade ruby, it should be separate item and PR

Reply:
Asana item: https://app.asana.com/0/1204208070292844/1204828735662357/f

Also, if we gonna upgrade, Ruby 2.7 will be the version, since Lexicon and DataStore are using datum and they are using Ruby 2.7 version